db/migrate/20201201213228_add_marked_for_deletion_to_downloads.rb

```diff
@@ -0,0 +1,5 @@
+class AddMarkedForDeletionToDownloads < ActiveRecord::Migration[6.0]
+  def change
+    add_column :downloads, :marked_for_deletion, :boolean, default: false
```

Comment:
should we combine all these migrations into a single one for simplicity?

Reply:
or were separate ones created because they were copied from active_storage?

Reply:
`create_active_storage_tables.active_storage.rb` was from Rails. 

I just separated these out - combining seems minor